.github/workflows/gen.yaml

```diff
@@ -10,8 +10,11 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+    - name: check CPU architecture
+      shell: bash
+      run: cat /proc/cpuinfo && uname -a
```

Comment:
This step is for debugging?
We can add this on demand, so we don't have to include this step in the usual run.

Reply:
oops, you already removed this. thank you.